ci-operator/step-registry/sandboxed-containers-operator/env-cm/sandboxed-containers-operator-env-cm-commands.sh

```diff
@@ -0,0 +1,53 @@
+#!/bin/bash
+
+catsrcname="kataci-index"
+# TODO: should use configurable branch instead of 'devel'?
+podvm_img_url="https://raw.githubusercontent.com/openshift/sandboxed-containers-operator/devel/config/peerpods/podvm/"
+runtimeclass="kata"
+test_workload="kata"
+configmap_path="${SHARED_DIR:-$(pwd)}/env-cm.yaml"
+
+# TODO: still needed? 600 seconds will cause the step timeout?
+#echo "Giving a 10min stabilization time for AWS fresh 4.18 cluster before applying kataconfig as workaround for KATA-3451"
+#sleep 600
+
+if [[ "$TEST_RELEASE_TYPE" == "Pre-GA" ]]; then
+  # TODO: implement me.
+  echo "Apply catalog source. Not implemented."
+  exit 1
+else
+  catsrcname="redhat-operators"
+fi
+
+if [[ "$ENABLEPEERPODS" == "true" ]]; then
+  runtimeclass="kata-remote"
+  # TODO: figure how to to set workload "coco" when CoCo testing
```

Comment:
coco: everything is the same as peer-pods, except the workloadToTest is "coco" instead of "peer-pods"

g.BeforeEach() will see coco and install trustee and configure it.  All the tests will see enablePeerPods and run kata-remote.  The pods/deployment will use a signed image instead of an unsigned one.

Reply:
Hi @tbuskey !

Cool, thanks for the explanation!

We will need to change that shell script to acomodate the coco case then. Because it's guessing the `workloadToTest` from `runtimeclass` but peer-pods and coco use same runtime. We may need to pass the workload to test as an parameter to the step (i.e. a environment variable) if there isn't a way to figure it out at the time the script runs.

Reply:
That's why osc-config has `workloadToTest`
I think going forward, the prowjob should have that (and pass it forward to osc-config)

workloadToTest:
- kata - runtimeClassName=kata, enablePeerPods=false
- peer-pods - runtimeClassName=kata-remote, enablePeerPods=true
- coco - runtimeClassName=kata-remote, enablePeerPods=true and install/configure trustee
-